hail/python/hail/methods/statgen.py

```diff
@@ -2140,8 +2567,19 @@ def skat(key_expr,
         Table of SKAT results.
 
     """
-    if hl.current_backend().requires_lowering and not logistic:
-        ht = hl._linear_skat(key_expr, weight_expr, y, x, covariates, max_size, accuracy, iterations)
+    if hl.current_backend().requires_lowering:
```

Comment:
In the second phase, we'll benchmark and address speed problems. Until we're certain it is fast enough, I don't want to enable it for all backends.